### PR TITLE
Support android 1.0

### PR DESCRIPTION
**Pull request by Ammar64**
I've also changed the green background of the icon on F-Droid  
I've made a Legacy.kt contains the implementations of functions not supported by android 1.0  
please test the youtube thing as youtube is blocked on my phone

### CODE REVIEW

**Comment by gabefair** [comment]
WOW! Thank you so much!!

**Comment by Ammar64** [comment]
Now your app supports every single android version on Earth.

File: Share2ArchiveToday/app/build.gradle.kts [hunk 1]
```diff
@@ -10,7 +10,7 @@ android {
 
     defaultConfig {
         applicationId = "org.gnosco.share2archivetoday"
-        minSdk = 19 // Updated to support Android KitKat (API 19)
+        minSdk = 1 // Updated to support Android KitKat (API 1)
```
Comment (Ammar64) [inline Share2ArchiveToday/app/build.gradle.kts:13]: I should've changed this comment.

Comment (gabefair) [reply Share2ArchiveToday/app/build.gradle.kts:13]: Fixed.  :)

**Pull request merged**